CHANGELOG_PENDING.md

```diff
@@ -29,6 +29,10 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 ### IMPROVEMENTS
 
+[cli] \#9171 add `--hard` flag to rollback command (and a boolean to the `RollbackState` method). This will rollback
+  state and remove the last block. This command can be triggered multiple times. The application must also rollback
+  state to the same height. (@tsutsu, @cmwaters)
```

Comment:
this indentation seems suspect.

Reply:
How would you prefer a multi-line changelog entry to look like?